src/coprocessor/dag/executor/mod.rs

```diff
@@ -177,6 +179,7 @@ pub fn build_exec(
                 src,
             )?),
             ExecType::TypeLimit => Box::new(LimitExecutor::new(exec.take_limit(), src)),
+            _ => unimplemented!(),
```

Comment:
Why unimplemented?

Reply:
@AndreMouche

Reply:
Since other types(TypeStreamAgg..) would be implemented in the later PR.

Reply:
please add the comment here.